db/migrate/20160308100307_add_filtered_api_to_ovirt.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilteredApiToOvirt < ActiveRecord::Migration
+  def change
+    add_column :compute_resources, :filtered_api, :boolean, :default => false, :null => false
```

Comment:
I can, I don't want to :-) especially in this table I don't expect thousands of records and I think the number of downsides it brings is too big (migration touching serialized data, possible inconsistency between two objects of same class, faulty implementation that uses method_missing without defining respond_to?)